entity-framework/toc.yml

```diff
@@ -34,30 +34,28 @@
           href: core/what-is-new/ef-core-5.0/whatsnew.md
         - name: Breaking changes
           href: core/what-is-new/ef-core-5.0/breaking-changes.md
-      - name: EF Core 3.0
+      - name: EF Core 3.x
         items:
         - name: New features
-          href: core/what-is-new/ef-core-3.0/index.md
+          href: core/what-is-new/ef-core-3.x/index.md
         - name: Breaking changes
-          href: core/what-is-new/ef-core-3.0/breaking-changes.md
-      - name: EF Core 2.2
-        href: core/what-is-new/ef-core-2.2.md
+          href: core/what-is-new/ef-core-3.x/breaking-changes.md
       - name: EF Core 2.1
         href: core/what-is-new/ef-core-2.1.md
-      - name: EF Core 2.0
-        href: core/what-is-new/ef-core-2.0.md
-      - name: EF Core 1.1
-        href: core/what-is-new/ef-core-1.1.md
-      - name: EF Core 1.0
-        href: core/what-is-new/ef-core-1.0.md
-      - name: Upgrade from previous versions
+      - name: Unsupported versions
```

Comment:
bit of nit but unsupported is different from no longer supported.

Reply:
OK, how about just "out of support"?

Reply:
Out of support works.